boilerplate/package.json

```diff
@@ -11,6 +11,9 @@
     "npm": ">=3.10.10"
   },
   "dependencies": {
-    "zapier-platform-core": "CORE_PLATFORM_VERSION"
+    "async": "2.6.0",
+    "moment-timezone": "0.5.16",
+    "zapier-platform-core": "CORE_PLATFORM_VERSION",
+    "zapier-platform-legacy-scripting-runner": "1.2.0"
```

Comment:
Would it be possible to have legacy-scripting-runner, `moment-timezone`, and `async` be part of the default lambda app we upload, vs being in core itself @eliangcs @BrunoBernardino?

Reply:
I'm not sure I understand the question. This will be the default lambda app we upload, it's not `core` itself (notice it's `boilerplate/package.json`, not `package.json`).

Sorry if I'm misunderstanding the question here!

Reply:
Not sure if I understand it correctly. Correct me if I misunderstood.

This `boilerplate/package.json` file **is** the package.json for the default lambda app we upload. The only reason why it's in `core` is that we build such a default lambda app in `core` ([PR](https://github.com/zapier/zapier-platform-core/pull/81)). The `core` itself doesn't require `legacy-scripting-runner`, `moment-timezone`, etc.

Reply:
Sorry, I commented in the wrong spot, but if you look at [the diff for package.json](https://github.com/zapier/zapier-platform-core/pull/86/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2) it looks like all those leaked into core itself, right?

This may deserve a real-time discussion, but I'm wondering if there is a way to not pollute core with code from a converted app. Is there a way to separate concerns here?

Reply:
It's a part of `devDependencies`, as `core` builds `boilerplate/X.zip` on release (it doesn't get into apps).

Reply:
@bcooksey another reason why they're in `devDependencies` is that we have legacy scripting tests in `core` [here](https://github.com/zapier/zapier-platform-core/blob/649866eafeaaea0edbf196a40425bbcf8c1f7bcf/test/legacy-scripting/legacy-scripting.js). I actually thought about moving all of these (`moment-timezone`, `async`, etc) to `legacy-scripting-runner` so that `core` doesn't need to know about the existence of `legacy-scripting-runner`. But that didn't work because we need to [create a z-app](https://github.com/zapier/zapier-platform-core/blob/649866eafeaaea0edbf196a40425bbcf8c1f7bcf/test/legacy-scripting/legacy-scripting.js#L34) to run those tests, which can't be done in `legacy-scripting-runner`. It's a bit hacky, but I wish I had a better solution.

Reply:
Got it, makes sense!